frontend/src/components/Dashboard.vue

```diff
@@ -161,50 +208,71 @@ export default {
       const labels = [];
       const dataset = [];
       for (var key of response.data.entries()) {
-        labels.push(key[1]["IDP"].padEnd(20))
-        dataset.push(key[1]["UNIQUE_USER_COUNT"])
+        labels.push(key[1]["IDP"].padEnd(20));
+        dataset.push(key[1]["UNIQUE_USER_COUNT"]);
       }
 
       this.uniqueUserCountByIDP["labels"] = labels;
       this.uniqueUserCountByIDP["UNIQUE_USER_COUNT"] = dataset;
       this.uniqueUserCountByIDPLoadingStatus = false;
     },
     async loadUniqueUserCountByRealm() {
-      const response = await MetricsRepository.get("unique-user-count-by-realm");
+      const response = await MetricsRepository.get(
+        "unique-user-count-by-realm"
+      );
       const labels = [];
       const dataset = [];
       for (var key of response.data.entries()) {
-        labels.push(key[1]["REALM"].padEnd(20))
-        dataset.push(key[1]["UNIQUE_USER_COUNT"])
+        labels.push(key[1]["REALM"].padEnd(20));
+        dataset.push(key[1]["UNIQUE_USER_COUNT"]);
       }
 
       this.uniqueUserCountByRealm["labels"] = labels;
       this.uniqueUserCountByRealm["UNIQUE_USER_COUNT"] = dataset;
       this.uniqueUserCountByRealmLoadingStatus = false;
     },
+    async loadActiveTotalUser(format) {
+      this.totalUserCountSelectedFormat = format;
+      this.totalUserCountLoadingStatus = true;
+      const response = await MetricsRepository.get(
+        "total-active-user-count/" + format
+      );
+      const labels = [];
+      const dataset = [];
+      for (var key of response.data.entries()) {
+        labels.push(new Date(key[1]["DATEE"]).toISOString().split("T")[0]);
+        dataset.push(key[1]["ACTIVE_USER_COUNT"]);
+      }
+      this.totalUserCount["DATEE"] = labels;
```

Comment:
As per my comment from `MetricController.java`  the DATEE should be more descriptive